Core/src/org/sleuthkit/autopsy/geolocation/datamodel/GeolocationUtils.java

```diff
@@ -0,0 +1,199 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.sleuthkit.autopsy.geolocation.datamodel;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.sleuthkit.datamodel.BlackboardArtifact;
+import org.sleuthkit.datamodel.BlackboardAttribute;
+import org.sleuthkit.datamodel.TskCoreException;
+
+/**
+ *
+ *
+ */
+public final class GeolocationUtils {
```

Comment:
I don't think this class hangs together very well. A bunch of the methods are more like artifact utilities which have nothing specific to geolocation data; I thought we talked about putting them in their own class in case we wanted to move them down to TSK? Then, setting aside the unused method (see below), that leaves us with getOtherGeolocationAttributes used by the Wayppoint implementations, and getFormattedCoordinates, for which the KML report is the only client, although that may change. Thus we have a client utility mixed with a helper for the geolocation data model classes. This is a classic issue when putting together broad utility classes...

Reply:
I don't think we ever came to a firm conclusion about what exactly to do with those functions.  I don't disagree that this became a dump for stuff I didn't know where else to put.

The getFormattedCoordinates was put there the thought of consistency between formatting with clients that may use this, but really it feels more like it belongs in the report.

Let's talk more about this one.

Reply:
Sounds good, we can talk it over in person today.